vnext/Microsoft.ReactNative/Base/CoreUIManagers.cpp

```diff
@@ -32,6 +33,7 @@ void AddStandardViewManagers(
     std::vector<std::unique_ptr<Microsoft::ReactNative::IViewManager>> &viewManagers,
     const Mso::React::IReactContext &context) noexcept {
   viewManagers.push_back(std::make_unique<ActivityIndicatorViewManager>(context));
+  viewManagers.push_back(std::make_unique<DebuggingOverlayViewManager>(context));
```

Comment:
If you look below, Tatiana added a new parameterized `UnimplementedViewManager` we can use to stub view managers without having to create a fully stubbed out class.

Reply:
UnimplementedViewManager would still cause a redbox to occur when using react-devtools.  This also provides most of the code we'll want when we actually go to implement DebugggingOverlay.

Reply:
The idea was, as more stuff gets added in Fabric that we _aren't_ ever going to ever add in Paper, we have an easy reusable ViewManager for that.

If we are planning on implementing this component eventually, fine, but if the only problem is that our current unimplemented viewmanager redboxes instead of no-ops, the maybe we just need a flag so UnimplementedViewManager can use the behavior we want.